applications/ContactStructuralMechanicsApplication/tests/ALM_frictionless_contact_test_2D/mesh_moving_notmatching_test_results.json

```diff
@@ -1 +1 @@
546]}, "NODE_35": {"DISPLACEMENT_X": [-0.023351335822573503, -0.011819247903972393, 0.11891661299960549, 0.2509692507014469, 0.38182117768141416, 0.5126857248040194, 0.6448995172354995, 0.7715949624282719, 0.9064758168025357, 0.9157306441491854, 0.784439988889382, 0.6546196606786434, 0.521951265146601, 0.3906314210153212, 0.26072778189285273, 0.12817838059200387, -0.0031751796383950623], "DISPLACEMENT_Y": [-0.035821181219360676, -0.035701887382857865, -0.03606000652568337, -0.037727460349501814, -0.035991626274164384, -0.03621945648217362, -0.03801311083551449, -0.035630084175500246, -0.03617697496203797, -0.0364893605245165, -0.03626769105776108, -0.03813477039353476, -0.03652773822098013, -0.036218864424809, -0.03787776576575251, -0.036370716518591785, -0.035925630837413014]}, "NODE_39": {"DISPLACEMENT_X": [0.0022590840397993302, 0.002301336188609632, 0.0019392413309041508, 0.001406891322463076, 0.000598634717107964, -0.00010899540734087613, -0.0007248899965388344, -0.00121943954105077, -0.001553442449398659, -0.0016310897882338316, -0.0013114146312869731, -0.0008513739509286736, -0.00023638043665272714, 0.0004646218730999802, 0.0012163281753755765, 0.0017646601453882485, 0.002144782891942464], "DISPLACEMENT_Y": [-0.02237189206226016, -0.021300517991631792, -0.012101952830004396, -0.005193326855331587, -0.0027672151812652467, -0.0010218385753459707, 0.0002825474227859695, 0.0007958158163943851, 0.0010644528992526426, 0.001085190270409533, 0.0008253645627897341, 0.00032393851337854206, -0.0009458559512559176, -0.002667721598051876, -0.00507316246073311, -0.011745157785848141, -0.020793708428616258]}, "NODE_45": {"DISPLACEMENT_X": [0.0015933045196590524, 0.0015556446554838592, 0.0009153581580795878, 0.0003284082085598653, -0.00019938146508316325, -0.0005753482644627642, -0.0008627548337711778, -0.000988761650277525, -0.0010519284281549023, -0.001084505311085592, -0.0010285771169496554, -0.0009170636146273136, -0.0006348813534886479, -0.0002652576620199583, 0.00023339758060775232, 0.0008210326488323616, 0.0014653655869448065], "DISPLACEMENT_Y": [-0.002231468399327593, -0.0020397848849805986, -0.00038064078232221435, 0.0008196165077470075, 0.0012279863050812696, 0.0013996189167086413, 0.001460221964200122, 0.0012848300886438243, 0.001109038791897729, 0.0011134988098398608, 0.0012933588318270127, 0.00147092288497016, 0.0014199124467222054, 0.0012555890656368162, 0.0008621068282691907, -0.00030272839491851227, -0.0019399885570926916]}, "NODE_48": {"DISPLACEMENT_X": [0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0], "DISPLACEMENT_Y": [0.0012269474934462879, 0.0012979233675630033, 0.001848103087485157, 0.002141565701608695, 0.001937831430995609, 0.0016927630471475807, 0.0014490083550660914, 0.0011914925466656248, 0.0009717058200175363, 0.0009744197600855788, 0.0011972506844963227, 0.001455896561210448, 0.0016986145019140262, 0.0019435822859551017, 0.0021568582136395615, 0.0018859595692014626, 0.0013502642510822246]}}
```

Comment:
why do the results change if you only refactor?

Reply:
I changed some defaults